_includes/header--help.html

```diff
@@ -2,7 +2,7 @@
   <div class="usa-nav-container">
     <div class="usa-navbar">
       <div class="usa-logo">
-        <a href="{{ '/' | locale_url }}" title="Home" aria-label="Home">
+        <a href="{{ '/' | locale_url }}" aria-label="Home">
```

Comment:
maybe for future, should we worry about translating this label? (and in `header.html` too)

Reply:
> maybe for future, should we worry about translating this label? (and in `header.html` too)

Late follow-up, but I just noticed this comment and created a ticket: [LG-12005](https://cm-jira.usa.gov/browse/LG-12005)